test/functional/eval/null_spec.lua

```diff
@@ -42,14 +42,8 @@ describe('NULL', function()
   describe('list', function()
     -- Incorrect behaviour
 
-    -- FIXME map() should not return 0 without error
-    null_expr_test('does not crash map()', 'map(L, "v:val")', 0, 0)
-    -- FIXME map() should not return 0 without error
-    null_expr_test('does not crash filter()', 'filter(L, "1")', 0, 0)
-    -- FIXME map() should at least return L
-    null_expr_test('makes map() return v:_null_list', 'map(L, "v:val") is# L', 0, 0)
-    -- FIXME filter() should at least return L
-    null_expr_test('makes filter() return v:_null_list', 'map(L, "1") is# L', 0, 0)
+    null_expr_test('makes map() return v:_null_list', 'map(L, "v:val") is# L', 0, 1)
+    null_expr_test('makes filter() return v:_null_list', 'map(L, "1") is# L', 0, 1)
```

Comment:
Here is a copy-paste typo, test needs to test filter(). And, BTW, list with OK tests is *below*, in this part only tests that need fixing are located. You may see “Incorrect behaviour” comment above.